resources/campaigns/campaign.yml

```diff
@@ -0,0 +1,114 @@
+parameters:
+  - in: path
+    name: cmp_id
+    description: id of the campaign
+    required: true
+    schema:
+      $ref: "attributes/cmp_id.yml"
+
+get:
+  operationId: campaign_retrieve
+
+  summary: Retrieve
+
+  description: >-
+    Retrieves the details of an existing campaign. You need only supply
+    the unique campaign identifier that was returned upon campaign creation.
+
+  tags:
+    - Campaigns
+
+  responses:
+    "200":
+      description: Returns a campaign object
+      content:
+        $ref: "responses/campaign.yml"
+
+    default:
+      $ref: "responses/campaigns_error.yml"
+
+  x-codeSamples:
+    - lang: Shell
+      source: |
+        curl https://api.lob.com/v1/campaigns/cmp_e05ee61ff80764b \
+          -u <YOUR API KEY>:
+      label: CURL
+
+patch:
+  operationId: campaign_update
+
+  summary: Update
+
+  description: >-
+    Update the details of an existing campaign. You need only supply
+    the unique identifier that was returned upon campaign creation.
+
+  tags:
+    - Campaigns
+
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: "models/campaign_updatable.yml"
+        example:
+          # TODO: fill out example
```

Comment:
We have a few `Todo's` here. Do we need examples before merging, or is there a specific JIRA task we can include so this doesn't get lost in the shuffle